store/server/features/approver/mutation.ts

```
@@ -148,6 +148,13 @@ export const useCreateApproverMutation = () => {
         description: 'Approval WorkFlow Created Successfully',
       });
     },
+    onError: (error: any) => {
+      const errorMessage = error?.response?.data?.message || 'Something went wrong';
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)